setup.py

```diff
@@ -11,7 +11,8 @@
 
 # Get version and release info, which is all stored in datarray/version.py
 ver_file = os.path.join('datarray', 'version.py')
-execfile(ver_file)
+# Use exec on contents for Python 3 compatibility
+exec(open(ver_file).read())
```

Comment:
For completeness, probably better to do as a [context manager](http://stackoverflow.com/a/16515875).